cli/command/service/rollback.go

```diff
@@ -0,0 +1,65 @@
+package service
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/docker/api/types"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func newRollbackCommand(dockerCli command.Cli) *cobra.Command {
+	options := newServiceOptions()
+
+	cmd := &cobra.Command{
+		Use:   "rollback [OPTIONS] SERVICE",
+		Short: "Rollback a service",
+		Args:  cli.ExactArgs(1),
```

Comment:
For consistency with other commands, it would be good to support multiple services but let's leave that as a follow-up as it may need further discussion